tests/tests/item.test.js

```diff
@@ -0,0 +1,105 @@
+const request = require("supertest");
+
+const { createApp } = require("../../app");
+const dataSource = require("../../api/models/dataSource");
+const testClient = require("../test-client");
+const usersFixture = require("../fixtures/users-fixture");
+const itemsFixture = require("../fixtures/items-fixture");
+const categoriesFixture = require("../fixtures/categories-fixture");
+const materialsFixture = require("../fixtures/materials-fixture");
+const itemsMaterialsFixture = require("../fixtures/items-materials-fixture");
+
+describe("Read Item Details", () => {
+  let app;
+
+  const tables = [
+    "users",
+    "items",
+    "categories",
+    "materials",
+    "items_materials",
+  ];
+
+  const user = {
+    id: 1,
+    email: "changsup96@naver.com",
+    name: "이창섭",
+    password: "changsup96",
+    providerId: "2186574589",
+    bankName: "국민",
+    bankAccount: "102-987-456254",
+    artistRegistration: "IGA-421",
+    bidAgreement: true,
+  };
+
+  const category = {
+    id: 1,
+    name: "회화",
+  };
+
+  const material = {
+    id: 1,
+    name: "고해상도흑백카메라",
+  };
+
+  const item = {
+    id: 1,
+    sellerId: 1,
+    categoryId: 1,
+    authorName: "진코딩",
+    itemName: "다누리가 찍은 달",
+    productionYear: 2023,
+    width: 100.0,
+    length: 100.0,
+    height: 3.0,
+    weight: 3.0,
+    adminNumber: "AVED951351",
+    description:
+      ' "한국산 달 인공위성인 다누리가 달 주변을 돌며 찍어 보낸 사진입니다."',
+    startingBid: 1351300.0,
+    imageUrl: "https://wevre.s3.ap-northeast-2.amazonaws.com/Items/20.avif",
+    biddingStart: "2023-03-28 00:00:00",
+    biddingEnd: "2023-04-04 00:00:00",
+  };
+
+  const itemMaterial = {
+    id: 1,
+    itemId: 1,
+    materialId: 1,
```

Comment:
```suggestion
    itemId: item.id,
    materialId: material.id,
```
위에서 남겨드린 코멘트와 같은 맥락으로, 지금은 단순 item, material이긴 하지만 이름이 명확하다면 구체적으로 어떤 item의 id인지, 어떤 material의 id인지 쉽게 파악할 수 있습니다.